ci/sim_install.sh

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+pip install --user toml
```

Comment:
I wonder if it'd be useful to add this to the (or a) requirements file?

Reply:
Can do.